### ISSUE TITLE
Grading Criteria Metadata

### ISSUE THREAD

**Issue by millerg23** [pharmaverse/admiral#2557]
Grading Criteria Metadata NCI-CTCAEv5

Some CTCAE grade definitions contain two sets of units, but only one is provided in the current grading metadata. For example, TERM=Hypoalbuminemia has only one row for unit = g/L, so it's not be able to grade for unit = g/dL
can we add more rows to the metadata to cover different units?

Originally posted by @kaz462 in https://github.com/pharmaverse/admiral/discussions/2191#discussioncomment-8074207

**Comment by bms63** [comment]
@millerg23 does this still need to be addressed?

**Comment by bms63** [comment]
Hey @millerg23 did this get updated in another Pull Request?

**Comment by millerg23** [comment]
I will try to get some time to do this for next release (1.3).
(We split original ticket into 2 separate issues, we addressed the BASELINE abnormality criteria in 1.2, this one was scheduled for 1.3).

**Comment by manciniedoardo** [comment]
@millerg23 is this possible to do prior to 1.3?

**Comment by millerg23** [comment]
I will take a look over next couple of days.

**Comment by millerg23** [comment]
Yes, I can get this done. Definitely for NCICTCAEv5 and v4, and probably DAIDs (time allowing).
